src/server/models/education.model.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+import mongoose from 'mongoose';
+
+const Schema = mongoose.Schema;
+let educationSchema = new Schema({
+    schoolName: String,
+    degree: String,
+    state: String,
+    startDate: Number,
```

Comment:
How will you represent date? In unix epoch time? Does it make sense to use the Date type?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date

I don't know the right answer: in any case, leave a TODO comment to resolve this question at a later date.